src/lib/wc.js

```diff
@@ -5,7 +5,7 @@ const wc = data => {
 
   const rows = data.split('\n')
```

Comment:
This would break since `\n` is not necessarily the line splitter. It could be \r\n or even any other fancy thing.

I guess you could just ask for an array of strings / rows.

Reply:
What `fancy thing`?

It is a CSV library, lines will be separated by `\n`. Not sure about `\r\n`; so may be pass rows.

Reply:
> What `fancy thing`?

I dunno. Just a guess.

Reply:
So rows it is.

Reply:
😞